misc/autodoc/autodoc.py

```diff
@@ -162,8 +160,11 @@ def add_page(title, text):
     add_page(overview_title, overview_text)
     return changed_pages
 
+
 if __name__ == '__main__':
     args = parse_args()
+    if not args.dry_run and PASSWORD is None:
+        sys.exit(f"{ENV_VAR_PASSWORD} not set.")
```

Comment:
This should follow the conventions (using `logging`, then `sys.exit(1)`) used in the rest of the file.

Reply:
Specifically, I would go with `logging.critical`, as in the old code that read the password from the file.

Reply:
I dislike how some comments simply disappear in the diff-GUI, because the code section was changed...

I have incorporated your comment.